src/main.rs

```diff
@@ -148,8 +148,8 @@ impl Actor for Node {
 
 #[derive(Clone)]
 struct ModelCfg {
-    elder_count: usize,
-    server_count: usize,
+    genesis_nodes: usize,
+    final_nodes: usize,
```

Comment:
How about just
```suggestion
    nodes: usize,
```

Reply:
Yeah. This place is the first that was confusing me.

What I am trying to make clear, is what got clear when this was explained to me, that one value is the _initial value_, and one is the _final value_. We start here, and aim to reach here. Two ends of a time line. Not two dots along a timeline.
`genesis_nodes` and `nodes` still makes me wonder if this is some distinction between different types of nodes that will be running in the simulation.

Reply:
`nodes` isn't a target value though, it's "How many nodes will be in the system"

Reply:
Yeah, I don't  mean it as  target value.

"will be" is what is meant with "final". We set what will be, the final value, through setting the number there.
We start with n elder and there will be (final value) k nodes.

So "final" reflects "will be" (those are quite synonymous wouldn't you say?)